msal/src/telemetry/java/com/microsoft/identity/client/ApiEvent.java

```diff
@@ -44,7 +44,9 @@ private ApiEvent(Builder builder) {
         if (null != builder.mRequestId) {
             setProperty(EventProperty.REQUEST_ID, builder.mRequestId);
         }
-        setProperty(EventProperty.AUTHORITY_NAME, HttpEvent.sanitizeUrlForTelemetry(builder.mAuthority));
+        if (!MsalUtils.isEmpty(builder.mAuthority)) {
```

Comment:
This isn't strictly speaking true: calls to `PublicClientApplication#getUsers()` are not associated with an authority yet still create `ApiEvents`. Yes, the `PublicClientApplication` will have `mAuthorityString` initialized, but the result of this call isn't guaranteed to return results related to that authority.